challenge_spr/src/challenge_spr_states/riddle_game.py

```diff
@@ -1,207 +1,215 @@
 #!/usr/bin/python
 
-import roslib;
+import roslib
```

Comment:
Do we actually need roslib? And the shebang?

Reply:
Not anymore, since ages, I think. Seems like a copy paste error for ye olde code